examples/fastspeech2/preprocess.py

```diff
@@ -0,0 +1,157 @@
+# Given the path to ljspeech/wavs,
+# this script converts wav files to .npy features used for training.
+
+import argparse
+import os
+from multiprocessing import cpu_count
+
+import mindspore as ms
+import numpy as np
+import pyworld as pw
+from dataset import all_dirs, all_postfix, feature_columns
+from mindspore.dataset.audio import MelScale, Spectrogram
+from phonemes import get_alignment
+from tqdm import tqdm
+
+import mindaudio
+from mindaudio.data.io import read
+from ljspeech import LJSpeech, create_ljspeech_tts_dataset
+from text import text_to_sequence
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--device_target", "-d", type=str, default="CPU", choices=("GPU", "CPU", "Ascend")
+)
+parser.add_argument("--device_id", "-i", type=int, default=0)
+parser.add_argument(
+    "--config",
+    "-c",
+    type=str,
+    default="recipes/LJSpeech/tts/fastspeech2/fastspeech2.yaml",
+)
+args = parser.parse_args()
+hps = mindaudio.load_config(args.config)
```

Comment:
same as above load config function. please upload your config.py to examples/fastspeech2